### PR TITLE
dirfd,xattrs: Port mostly to new code style

### PR DESCRIPTION
**Pull request by cgwalters**
Not everything, but a good chunk of the remaining bits.

### CODE REVIEW

**Comment by gnomesysadmins** [comment]
Thank you for contributing to libglnx!

libglnx uses Bugzilla for code review.

If you have never contributed to GNOME before make sure you have read the
getting started documentation:
http://www.gnome.org/get-involved

Otherwise please visit
https://wiki.gnome.org/Newcomers
and follow the instructions there to upload your change to Bugzilla.

File: glnx-xattrs.c [hunk 1]
```diff
@@ -270,31 +270,23 @@ set_all_xattrs_for_path (const char    *path,
...
 {
-  gboolean ret = FALSE;
-  int i, n;
-
-  n = g_variant_n_children (xattrs);
-  for (i = 0; i < n; i++)
+  const guint n = g_variant_n_children (xattrs);
+  for (guint i = 0; i < n; i++)
     {
+
```
Comment (jlebon) [inline glnx-xattrs.c:276]: Minor nit: extra space.

**Review by jlebon — COMMENTED** [review]
👍  Just a super minor nit.

File: glnx-xattrs.c (hunk 1 above)
Comment (cgwalters) [reply glnx-xattrs.c:276]: Thanks, fixed!

**Pull request closed without merging**